src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -8,7 +8,7 @@
 import { Setup as CoreSetup } from '@salesforce/lwc-dev-mobile-core/lib/cli/commands/force/lightning/local/setup';
 
 export class Setup extends CoreSetup {
-    public async run(): Promise<any> {
```

Comment:
Had forgotten to add the extra method signature. Good thing we're adding more tests now 😆